shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -220,6 +274,44 @@ private void setTextInputEditingState(View view, TextInputChannel.TextEditState
     }
 
     private void clearTextInputClient() {
-        mClient = 0;
+        if (inputTarget.type == InputTarget.Type.PLATFORM_VIEW) {
+            // Focus changes in the framework tree have no guarantees on the order focus nodes are notified. A node
```

Comment:
Is there any particular reason that we don't guarantee order? Seems like an unfortunate behavior...

Reply:
Yeah @gspencergoog  and I discussed this and we plan to change that behavior to first notify nodes that lost focus.

With that being said we're both ok with landing this until the focus manager behavior changes.

Reply:
Sounds good.